extern/storage-sealing/sector_state.go

```diff
@@ -1,43 +1,43 @@
 package sealing
 
-type SectorState string
+type SectorState uint8
```

Comment:
Changing this will break everyones miner sector states (this is used in on-disk state). Using strings also makes it much easier to migrate states.

The correct fix for this issue is likely just having a list of valid states, and checking against that list in the update-state command